src/Tribe/Duplicate/Strategy/Like.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * Class Tribe__Duplicate__Strategy__Like
+ *
+ * Models a loose similarity strategy, punctuation is removed from string and words can be in any order.
+ *
+ * @since TBD
+ */
+class Tribe__Duplicate__Strategy__Like
+	extends Tribe__Duplicate__Strategy__Base
+	implements Tribe__Duplicate__Strategy__Interface {
+
+	/**
+	 * Returns a string suitable to be used as a WHERE clause in a SQL query.
+	 *
+	 * @param string $key
+	 * @param mixed  $value
+	 *
+	 * @return string
+	 *
+	 * @since TBD
+	 */
+	public function where( $key, $value ) {
+		/** @var wpdb $wpdb */
+		global $wpdb;
+
+		if ( $this->is_a_numeric_post_field( $key ) ) {
+			return $wpdb->prepare( "{$key} = %d", $value );
+		}
+
+		$frags = $this->get_where_frags( $value );
+
+		$where_frags = array();
+		foreach ( $frags as $frag ) {
+			$prepared = $wpdb->prepare( '%s', '%' . strtolower( trim( $frag ) ) . '%' );
+			$where_frags[] = "{$key} LIKE {$prepared}";
```

Comment:
This reads a little odd, maybe these two lines should be more like:

```php
$formatted_frag = '%' . strtolower( trim( $frag ) ) . '%';
$where_frags[] = $wpdb->prepare( "{$key} LIKE %s", $formatted_frag );
```

I'm also wondering if these need to get run through `$wpdb->esc_like()` too?

```php
$formatted_frag = '%' . $wpdb->esc_like( strtolower( trim( $frag ) ) ) . '%';
```